LICENSE

```diff
@@ -1,2 +1,996 @@
-Copyright (c) 2016 Massachusetts Institute of Technology
-All rights reserved.
+Copyright (c) 2015-2016 Massachusetts Institute of Technology
+
+This library is free software; you can redistribute it and/or modify it under
+the terms of the GNU Lesser General Public License as published by the Free
+Software Foundation; either version 2.1 of the License, or (at your option) any
+later version.
+
+This library is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License along
+with this library (copied below); if not, write to the Free Software Foundation,
+Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA.
+
+Other licensing options may be available if required; contact the MIT Technology
```

Comment:
@jefrench, is it ok by you to make this explicit like I do here? I figure that gives people a path forward if LGPL doesn't work for them for some reason. (I suspect TLO will contact you if someone does ever get in touch with them, btw.)